operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -934,6 +963,54 @@ public boolean equals(Object other) {
     return builder.isEquals();
   }
 
+  /**
+   * Check to see if the only change in the spec is introspectVersion and/or model.onlineUpdate.
+   * @param other - another DomainSpec object for comparison
+   * @return true if the change is only introspectVersion and/or model.onlineUpdate
+   */
+  public boolean isSpecChangeForOnlineUpdateOnly(Object other) {
+    if (other == this) {
+      return true;
+    }
+    if (!(other instanceof DomainSpec)) {
+      return false;
+    }
+
+    DomainSpec rhs = ((DomainSpec) other);
+    // Note: This builder excluded introspectVersion
+    // and configuration
+    EqualsBuilder builder =
+        new EqualsBuilder()
+            .appendSuper(super.equals(other))
```

Comment:
This looks a little fragile to me in terms of future maintenance - namely, how will someone that comes along later and adds new attributes know that they might need to modify this particular equals statement?  I don't have solution for this - maybe there isn't one - but, if there's an easy one you can think of?  For example, could each attribute have an annotation to indicate whether it belongs in this equals statement? @rjeberhard FYI

Reply:
This is the part that I am not too comfortable with and need some input?  Essentially when do we consider it is not supported change in domain resource for online update?

Reply:
There's a lot of overlap with the main 'equals' in this same file.  Perhaps they can share code? That would seem to make it less likely that they would get out of sync. @rjeberhard FYI

Reply:
Closing no longer applicable.